packages/api/src/external/commonwell/document/document-query.ts

```diff
@@ -553,7 +567,13 @@ export async function downloadDocsAndUpsertFHIR({
             },
           };
 
-          if (file.isNew && !ignoreFhirConversionAndUpsert) {
+          if (!shouldConvertCDA && isConvertibleDoc) {
+            errorCountConvertible++;
```

Comment:
To better represent what's going on here, couldn't we use `increaseCountConvertible--` and allow it to be negative on `appendDocQueryProgress`? Then we can also adjust [this other place](https://github.com/metriport/metriport/pull/946/files#diff-3854a322c805912c1edf625353f3a6ea044f1dc1e63c294da26bf424f5478ce4R482) where we currently use it but it's not an error.

If we do, I'd just add a `Math.min(0, value)` where we set `convert.total` - [here](https://github.com/metriport/metriport/pull/946/files#diff-17fbc8ded4f6fb19fb78ec228367f0f9a5db135a2cffde2d6d334f68a0f330ccR83).